src/havard_eig_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,32 @@
+import math
+
+
+def letter_freq(txt):
+    dictionary = {}
+    txt = txt.lower()
+    for j in txt:
+        if j in dictionary:
+            dictionary[j] += 1
+        else:
+            dictionary[j] = 1
+    return dictionary
+
+
+def entropy(message):
+    char_freq = letter_freq(message)
+    tot_char = 0
+    entropy_ = 0
+    for _ in message:
+        tot_char += 1
+    freq_letter = {}
+    for key, value in char_freq.items():
+        freq_letter[ord(key)] = value/tot_char
+    for value in freq_letter.values():
+        entropy_ += value * math.log2(value)
```

Comment:
You can combine counting entropy in the previous for loop by adding up the value after each iteration.